mantidproject/mantid
# Create Profiling dev-docs page and minor testing updates

**Description of work.**

Create profiling overview page
Changing instances of 'unscripted' to 'manual' testing
Removing instances of the word 'MantidPlot' from testing
Add links to Core and SliceViewer testing docs
Add ISIS facilities as another newstarter Dev Account
Removed the very outdated Meet the Team section

**To test:**


Build dev-docs-html and check altered pages and links.
Check the new profiling page follows the [docs guidelines](https://developer.mantidproject.org/Standards/DocumentationGuideForDevs.html#how-to-define-titles-sections-etc)

Fixes #29337

*This does not require release notes* because **documentation change**



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
